libs/geoservices/covid/src/lib/pages/maps/maps.component.html

```diff
@@ -0,0 +1 @@
+<router-outlet></router-outlet>
```

Comment:
Prettier should be adding newlines at the end of a file?

Reply:
forgot to run format document on code. I'll set it to run on save.